File: Groove.SP/Groove.SP.Core/Entities/ProductVerificationSettingModel.cs
using Groove.SP.Core.Models;

namespace Groove.SP.Core.Entities
{
    public class ProductVerificationSettingModel : Entity
    {
        public long Id { get; set; }

        public long BuyerComplianceId { get; set; }

        public BuyerComplianceModel BuyerCompliance { get; set; }

        public VerificationSettingType PreferredCarrierVerification { get; set; }

        public VerificationSettingType ProductCodeVerification { get; set; }

        public VerificationSettingType CommodityVerification { get; set; }

        public VerificationSettingType HsCodeVerification { get; set; }

        public VerificationSettingType CountryOfOriginVerification { get; set; }

        public bool IsRequireGrossWeight { get; set; }

        public bool IsRequireVolume { get; set; }
    }
}

File: Groove.SP/groove-sp-web/src/app/features/compliance/models/product-verification-setting.model.ts
import { VerificationSetting } from 'src/app/core';

export class ProductVerificationSettingModel {
    id: number;
    preferredCarrierVerification: VerificationSetting;
    productCodeVerification: VerificationSetting;
    commodityVerification: VerificationSetting;
    hsCodeVerification: VerificationSetting;
    countryOfOriginVerification: VerificationSetting;
    isRequireGrossWeight: boolean;
    isRequireVolume: boolean;
}